.github/workflows/promptflow-replay-test.yml

```diff
@@ -79,12 +79,31 @@ jobs:
       working-directory: ${{ env.testWorkingDirectory }}
       run: |
         gci env:* | sort-object name
-        $env:PYTHONPATH=""
         cp ${{ github.workspace }}/src/promptflow/dev-connections.json.example ${{ github.workspace }}/src/promptflow/connections.json
         python "../../scripts/building/run_coverage_tests.py" `
         -p promptflow `
         -t ${{ github.workspace }}/src/promptflow/tests/sdk_cli_test `
         -l eastus `
         -m "unittest or e2etest" `
         --coverage-config ${{ github.workspace }}/src/promptflow/tests/sdk_cli_test/.coveragerc
+    - name: Publish Test Results
```

Comment:
this one looks good, and I will recommend to separate this part to another PR and we can target merge that one fast first.

Reply:
Do this after we test pull_request_target complete.